packages/orbit-components/src/Accordion/AccordionSection/index.tsx

```diff
@@ -43,12 +44,19 @@ const AccordionSection = ({
             expandable={expandable}
             expandOnTileClick={expandOnTileClick}
             dataTest={dataTest}
+            ariaControls={slideId}
+            id={headerId}
           >
             {header}
           </SectionHeader>
         )}
 
-        <Slide maxHeight={height} expanded={isExpanded} id={slideId} ariaLabelledBy={slideId}>
+        <Slide
+          maxHeight={height}
+          expanded={isExpanded}
+          id={slideId}
+          ariaLabelledBy={expandOnTileClick ? headerId : undefined}
```

Comment:
I don't think ariaLabelledBy is correctly used here. We are basically saying that the whole expanded content is labeled by the header. Does that make sense? 🤔 I think we don't need it here, or do we?